src/editor/map/context_manager.cpp

```diff
@@ -915,7 +915,14 @@ void context_manager::load_map(const std::string& filename, bool new_context)
 
 	if(filesystem::ends_with(filename, ".cfg")) {
 		if(editor_controller::current_addon_id_ == "") {
-			editor_controller::current_addon_id_ = editor::initialize_addon();
+			// if no addon id has been set and the file being loaded is from an addon
+			// then use the file path to determine the addon rather than showing a dialog
+			if(editor_controller::current_addon_id_ == "") {
+				editor_controller::current_addon_id_ = filesystem::get_addon_id_from_path(filename);
+			}
```

Comment:
Sorry, just noticed the previous if already checks this...

```suggestion
			editor_controller::current_addon_id_ = filesystem::get_addon_id_from_path(filename);
```